### DISCUSSION TITLE
Consistently allow GitHub Apps as exceptions to branch protection rules

### DISCUSSION THREAD

**Discussion by gitfool** [community/community#16875] [Other Feature Feedback, Questions, & Ideas]
> Now, branch protections that previously only allowed exceptions for people and teams also support GitHub Apps.

Unless I'm missing something, app exceptions do not seem to be available to user-owned [repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories)?

**Comment by blz-ea** [comment]
Experiencing same with private repositories in an org
No GitHub Apps are found when searched

**Reply by rbrooks-kryptowire** [reply]
Ah, maybe this explains why I can't find out app (our organization, the repos, and the app are private)

**Reply by blz-ea** [reply]
Turns out, app should have Commit statuses, and Contents (or maybe just Contents ?) permissions.
It works with those permissions, private org, private repos, and private github app)
